### ISSUE TITLE
Support v2 options when creating cases

### ISSUE THREAD

**Issue by joeljunstrom** [Oktavilla/egree-ruby#7]
All new options when creating cases (including templateId and the new EventCallback structure)

https://app.egree.com/api#Createacase

**Comment by jawa** [comment]
Is the `CaseOptionsMapper` needed or should all options go into the `Case` model? It seems like there is no clear distinction between the case and the options from the API documentation. Or should the required fields be considered part of the core case and the others as options?

**Comment by joeljunstrom** [comment]
I guess we could make the distinction on the difference between what is sent wen creating a case and what is returned when receiving one from the api? (If any)

**Comment by joeljunstrom** [comment]
But if you feel that the distinction does not add clarity or is unnecessary feel free to change it

**Comment by jawa** [comment]
No, that sounds reasonable to have the response of getcase to act as the base of the case so to speak :-)
